components/ExamScheduleViewer.tsx

```
@@ -62,27 +84,27 @@ const ExamScheduleViewer = () => {
-            columnFilters={columnFilters}
-            onColumnFilterChange={handleColumnFilterChange}
-          />
-          <ExamTableBody
-            entries={filteredEntries}
-            hiddenCols={hiddenCols}
-            colWidths={colWidths}
-          />
-        </table>
+          <table
+            className="w-full border-collapse table-fixed user-select-none select-none"
+            role="grid"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<table>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)